src/main/java/spring/model/dto/TaskReportDTO.java

```diff
@@ -0,0 +1,18 @@
+package spring.model.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class TaskReportDTO {
+    private Long taskId;
+    private LocalDate completionDate;
```

Comment:
do you need to track creational and update date?

Reply:
I think its redurant due to TaskDTO has this fields, am I wrong?

Reply:
if you have it, ofc, but i don't see them. Actually it's a question to you : do you need this fields or no. If you think it's okay not to track - just ignore